react/AccordionItem/AccordionItem.js

```diff
@@ -0,0 +1,82 @@
+// @flow
+
+import React, { Component } from 'react';
+import classnames from 'classnames';
+import styles from './AccordionItem.less';
+import { ChevronIcon } from '../';
+
+type Props = {
+  children: React$Node,
+  title: React$Node,
+  id: string | number,
+  expanded?: boolean,
+  onClick?: Function,
+  className?: string
+};
+
+type State = {
+  expanded?: boolean
+};
+
+export default class AccordionItem extends Component<Props, State> {
+  static defaultProps = {
+    expanded: false
+  }
+
+  constructor(props: Props) {
+    super(props);
+    const { expanded } = this.props;
```

Comment:
This should be destructuring from `props` i believe
```suggestion
    const { expanded } = props;
```